src/main/java/gregtech/loaders/recipe/MachineRecipeLoader.java

```diff
@@ -957,4 +958,25 @@ private static void registerNBTRemoval() {
         ModHandler.addShapelessRecipe("fluid_jetpack_clear", SEMIFLUID_JETPACK.getStackForm(), SEMIFLUID_JETPACK.getStackForm());
 
     }
+    private static void ConvertHatchToHatch() {
+        for (int i = 0; i < FLUID_IMPORT_HATCH.length; i++) {
+            if(FLUID_IMPORT_HATCH[i] != null) {
+
+                ModHandler.addShapelessRecipe(FLUID_IMPORT_HATCH[i].getMetaName() + FLUID_IMPORT_HATCH[i].getTier(), FLUID_IMPORT_HATCH[i].getStackForm(), FLUID_EXPORT_HATCH[i].getStackForm());
+                ModHandler.addShapelessRecipe(FLUID_EXPORT_HATCH[i].getMetaName() + FLUID_EXPORT_HATCH[i].getTier(), FLUID_EXPORT_HATCH[i].getStackForm(), FLUID_IMPORT_HATCH[i].getStackForm());
```

Comment:
Names should probably be something like: `"fluid_hatch_ev_input_to_output"`, etc.